app/services/va_dot_gov_address_validator.rb

```diff
@@ -111,11 +111,17 @@ def get_distance_to_facilities(facility_ids:)
   # Gets a list of RO facility ids to geomatch with.
   #
   # @return            [Array<String>]
-  #   An array of RO facility ids that are in the same state as the veteran/appellant.
+  #   Array of all RO facility ids for Travel board appeals or RO facility IDs by state for all other appeal types
   def ro_facility_ids_to_geomatch
     # only match to Central office if veteran requested central office
     return ["vba_372"] if appeal_is_legacy_and_veteran_requested_central_office?
 
+    # Return the list of RO facility IDs
+    if appeal.current_hearing_request_type == :travel_board
+      Rails.logger.info("Travel Board Appeal Geomatching | Appeal ID: #{appeal.class.name}#{appeal.id}")
+      return RegionalOffice.facility_ids
```

Comment:
@nweinmeister, @sahalliburton, @odonnadieu 
```suggestion
      return RegionalOffice.ro_facility_ids
```
Slack is down at the moment, but I think that this will fix the problem we've been having with missing/invalid ros.
`facility_ids` - Returns all facility_ids and alternate_facility_ids
`ro_facility_ids` - Returns ONLY the facility_ids associated with ros.

`ro_facility_ids` is much closer to what we use below (`ro_facility_ids_for_state`) in function as well, which confirms this.

Reply:
I think we should commit this, but unfortunately it doesn't fix the issue. I'm leaning pretty strongly towards the idea that the list returned by the va.gov api has changed recently.